### DISCUSSION TITLE
Performance issues happening on single page when pages has grown alot

### DISCUSSION THREAD

**Discussion by Luciogi** [LinwoodDev/Butterfly#666] [General]
Here is file:
[butterfly.bfly.zip](https://github.com/LinwoodDev/Butterfly/files/15015668/butterfly.bfly.zip)
(remove .zip extension)

- When moving around on canvas
- removing stroke with path eraser tool

**Comment by CodeDoctorDE** [comment]
Hi, can you give me more context (current Butterfly version, your operating system, ...)?
It would be nice if you can post this as bug report issue instead, discussions aren't good to track.

**Reply by Luciogi** [reply]
I tested on PWA and android same behavior on both platform

**Reply by CodeDoctorDE** [reply]
okay, can you please make an issue for this? It would be easier to track for me

**Reply by Luciogi** [reply]
ok
